### ISSUE TITLE
Locate xmake install.exe location

### ISSUE THREAD

**Issue by Meowchestra** [Meowchestra/MidiEditor#5]
Not sure where install.exe is being created when using binarycreator with xmake. It doesn't seem to be in the packaging folder or anywhere in the build directory.

**Comment by jingkaimori** [comment]
Installer.exe is supposed to be located at `packaging/Installer.exe` path

**Comment by Meowchestra** [comment]
@jingkaimori I had github actions pull the entire working folder into a zip and it wasn't located anywhere there. Even recursively listed every file in every directory, couldn't find it. :(

**Comment by jingkaimori** [comment]
another step of installation is required to generate installer:
```
xmake install installer
```

**Comment by Meowchestra** [comment]
In the github workflow I'm assuming?

**Comment by jingkaimori** [comment]
no, you should add this command to workflow manually, because I'm tweaking online installer thus this command does not appears in my workflow

**Comment by jingkaimori** [comment]
New Online installer is released at https://github.com/jingkaimori/midieditor/releases/tag/untagged-aecf17baf3a0b05b5911 , and packaging ci is finished.
